test/studies/shootout/submitted/mandelbrot3.compopts

```diff
@@ -0,0 +1 @@
+--no-warnings
```

Comment:
I think you should update the README in this directory if notest'ing the test is no longer the right course of action

Reply:
It also might be good to add a Github action that verifies the only modifications to the submitted directory are the approved ones?

Reply:
I've updated the README to try and clarify when notesting is appropriate.

I think a major challenge to having a GitHub action that verifies modifications are OK is that the most obvious check would be to see that `./clbg-diff.py` continues to pass, yet the CLBG maintainer only updates the tarball of sources that the script compares against every so often*; so even when things actually are in sync, they sometimes appear not to be according to the script.  It's also difficult to imagine a script (or at least, it being worth the time to build such a script) that could automatically distinguish between cases that should be handled in different ways.

(`*` = it was such an update that led to me running the script and developing this PR today)

Reply:
Yeah.  If there was a way to get an "are you sure about these updates to this directory?" message rather than an angry red X, that'd be ideal.  It just seems like it's really easy to see the test failure in that directory and forget to follow the proper steps and it can be hard for performance team members to notice when that has happened